src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    @Override
+    public List<Excavator> get() {
+        List<Excavator> excavators = new ArrayList<>();
+        Excavator excavator1 = new Excavator();
+        Excavator excavator2 = new Excavator();
+        Excavator excavator3 = new Excavator();
```

Comment:
If the `Excavator` class has a constructor that requires parameters or if there are attributes that need to be initialized, you should provide those values here. Otherwise, these instances will be created with default values, which might not be intended.